README.md

```diff
@@ -1 +1,19 @@
-# tf_activity1
\ No newline at end of file
+# Creation of resource group using terraform
+ 
+# There are two prerequisites followed to do this
+# Firstly, create service principal which is actually created the resources
+# Make service principal as contributor access
+
+# Set the environmemtal variables as below:
+# ARM_TENANT_ID:e2c1cb94-7a4c-4480-883d-c74888cadb8a
+# ARM_SUBSCRIPTION_ID:aaa7b7a2-a32b-4a09-9145-4a16b9b769b9
+# ARM_CLIENT_ID:b91dac8a-4af5-4509-a207-7303c7df027b
+# ARM_CLIENT_SECRETE:raP8Q~Wu~T4j5-iKNWD2nySQQ7D.2j4MHE~GxaXY
```

Comment:
You should never have these environment variables as part of the code or comment, it is a security issue

If anyone is having these four variables, they can use your AZURE resources or they can create

Reply:
I have done the changes, please review the PR

Reply:
Looks good now